packages/s3-credentials-endpoint/tests/test-handler-with-edl-token.js

```diff
@@ -77,6 +74,13 @@ test.before((t) => {
   nock.enableNetConnect('localhost:80');
 });
 
```

Comment:
Update unit test to fix the following error due to lambda update.
Error (NodeError) {
code: 'ERR_SERVER_ALREADY_LISTEN',
message: 'Listen method has been called more than once without closing.',
} 
› startServer (node_modules/@vendia/serverless-express/src/index.js:138:17)
› promise (node_modules/@vendia/serverless-express/src/index.js:********2:9)
› Object.proxy (node_modules/@vendia/serverless-express/src/index.js:197:14)
› handler (index.js:231:30)
› tests/test-handler-with-edl-token.js:110:26